backend/substrapp/tests/views/tests_views_compositetraintuples.py

```diff
@@ -117,7 +117,7 @@ def test_compositetraintuple_list_filter_compute_plan_id(self):
         with mock.patch('substrapp.views.compositetraintuple.query_ledger') as mquery_ledger:
             mquery_ledger.return_value = compositetraintuple
             compute_plan_id = get_compute_plan_id(compositetraintuple)
```

Comment:
Shouldn't we rename 'compositetraintuple' to 'composite_traintuple' here as well?

Reply:
I agree that in the backend knowing when there needs to be an underscore when and there shouldn't be one is pretty hard. In this case it seems to be in line with variable names in similar files (see for example `tests_views_datamanager.py`)